cozy_management/weboob.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+'''
+    Weboob management
+'''
+
+from helpers import cmd_exec
+from shutil import rmtree
+from sys import path
+from os import path
+
+WEBOOB_REPO = "git://git.symlink.me/pub/weboob/devel.git"
+
+def get_weboob_version():
+    result = cmd_exec("weboob-cli --version")
+    if (not result['error']):
+        return result['stdout'][0].rstrip()
+
+    return None
+
+def update():
+    print "Updating weboob configuration"
+    result = cmd_exec("weboob-config update")
+    if (result['error']):
+        print result['stderr']
+        print "Update failed"
+    else:
+        print "Update succeeded"
+
+def install():
+    '''
+        Install weboob system-wide
+    '''
+    tmp_weboob_dir = "/tmp/weboob"
+
+    # Check that the directory does not already exists
+    while (path.exists(tmp_weboob_dir)):
+        tmp_weboob_dir += "1"
+
+    # Clone the repository
+    print "Fetching sources in temporary dir " + tmp_weboob_dir
+    result = cmd_exec("git clone " + WEBOOB_REPO + " " + tmp_weboob_dir)
+    if (result['error']):
+        print result['stderr']
+        print "Weboob installation failed: could not clone repository"
+        exit()
+
+    print "Sources fetched, will now process to installation"
+
+    # Launch the installation
+    result = cmd_exec("cd " + tmp_weboob_dir + " && ./setup.py install")
+    if (result['error']):
+        print result['stderr']
+        print "Weboob installation failed: setup failed"
+    else:
+        print result['stdout']
+
+    # Remove the weboob directory
+    rmtree(tmp_weboob_dir)
+
+    if (result['error']):
```

Comment:
This still refers to the `result` line 50, so there's probably something missing just above (`result = rmtree(...)`, maybe?)